docs/source/tutorials/cmip6-ecs.ipynb

```diff
@@ -0,0 +1,3690 @@
+{
```

Comment:
<p>I think we could simplify/generalize this by adding a <code>keyfilter</code>  method to <code>xcollection.Collection</code>.   I imagine something along these lines:</p><p> </p><p><code>ecs_collection.keyfilter(lambda x: 'piControl' in x)</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds-funnel/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>